### ISSUE TITLE
Multilanguage support

### ISSUE THREAD

**Issue by MartinMeise** [Gifford47/HCPBridgeMqtt#86]
**Is your feature request related to a problem? Please describe.**
When I first opened the Web User Inferface it was all in englisch. So far so good. But localized would be even better.

**Describe the solution you'd like**

This Website describes in detail how to set up such a website: https://medium.com/@nohanabil/building-a-multilingual-static-website-a-step-by-step-guide-7af238cc8505

I could do the german translation.:
Garagentor
Tor geschlossen
Auf
Halb
Lüften
Stop
Zu
Licht
Basis Konfiguration einblenden
WLAN
WLAN Access Point aktivieren
WLAN SSID
WLAN Passwort
...
Speichern und Neustarten
Erweiterte Konfiguration

**Describe alternatives you've considered**

**Additional context**

**Comment by Gifford47** [comment]
Good point, but flash size is the limited factor here. We are almost at 70% of flash size used. so this request would be a "nice to have"
